puzzle.js

```diff
@@ -306,14 +313,22 @@ function PuzzleEntry(p) {
             this.undoManager.modifyAttribute(target, "value", text);
             this.undoManager.endGroup();
         }
+        this.processTdForCopyjack(target.parentElement); // TODO(jhimawan): fix after no-input refactor.
     }
 
     this.getText = function(target) {
         return target.value;
     }
 
     this.onUndoRedo = function(units) {
-        units.forEach((u) => { if (u.attribute == "data-path-code" || u.attribute == "data-edge-code" || u.attribute == "data-x-edge-code") { this.updateSvg(u.elem); }});
+        units.forEach((u) => { if (u.attribute == "data-path-code" || u.attribute == "data-edge-code" || u.attribute == "data-x-edge-code") {
+            this.updateSvg(u.elem);
+            if (u.elem instanceof HTMLTableCellElement) {
+                this.processTdForCopyjack(u.elem);
```

Comment:
If you did this outside of the attribute check, then you would not need to sprinkle copyjack calls around. The attribute checks are there to detect SVG cases, but all edits come through here...

Reply:
Also: I would like it to be done that way so that the copy support doesn't need to invade the entire codebase.

Reply:
Locally, I tried putting the `this.processTdForCopyjack(u.elem);` call outside of the attribute check and then deleting the rest of the references. But inputs and backgrounds (and possibly other things?) are no longer working. 

I agree we should try to put the copy support in as few places as possible, but I don't have a great understanding of the code flow. If you want to experiment on your end to figure out where to put the calls, that'd be very helpful!

Reply:
Update: I think I was high when I tried this initially... it seems to be working just fine now 🙃